packages/create-cloudflare/src/wrangler/__tests__/config.test.ts

```diff
@@ -71,6 +75,10 @@ describe("updateWranglerToml", () => {
 		const newToml = vi.mocked(writeFile).mock.calls[0][1];
 		expect(newToml).toMatch(`name = "${ctx.project.name}"`);
 		expect(newToml).toMatch(`main = "src/index.ts"`);
+		expect(newToml).toBe(
+			`compatibility_date = "2024-01-17"name = "test"\n` +
```

Comment:
is this supposed to be missing a newline between the first two lines?

Reply:
this is what https://github.com/cloudflare/workers-sdk/pull/7457 fixes and is not in the scope of this PR to fix.